### PR TITLE
Added Bower support

### PR DESCRIPTION
**Pull request by silviubogan**
- Improved a documentation comment according to a manual change in README.md;
- Changed version to 3.3.1 in package.json and source code and added me to the list of contributors;
- Added Bower support (bower.json);
- Started using node-blah for .gitignore (to which I added `bower_components`), CONTRIBUTING.md (a new file) and README.md (updated).

Solves #24.

:smiley: :smiley: :smiley: 

@IonicaBizau, what remains to be done, after merging this pull request, is to register the module in [the Bower registry](http://bower.io/search/) using [this](http://bower.io/docs/creating-packages/#register). Should you or I do this? (And of course, what remains to be done is to create a new tag with the new version.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,36 @@
...
+<!--                                                                        -->
+<!-- If you have any other ideas, open an issue.                            -->
+<!--                                                                        -->
+<!-- Please consider reading the contribution steps (CONTRIBUTING.md).      -->
+<!-- * * * Thanks! * * *                                                    -->
+<!---------------------------------------------------------------------------->
+
+# jquery-sidebar
 
-[![](http://i.imgur.com/4BLylrK.jpg)](http://jillix.github.io/jQuery-sidebar/)
```
Comment (IonicaBizau) [inline README.md]: Am I wrong or this just :fire: the image?

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "node_modules",
+    "bower_components"
+  ],
+  "dependencies": {
+    "jquery": "*"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jillix/jQuery-sidebar.git"
+  }
```
Comment (IonicaBizau) [inline bower.json]: You have to add here:

```json
{
    "blah": {
        "ex_img": "<screenshot-url>"
    }
}
```

Comment (IonicaBizau) [inline bower.json]: Ah, wait, not here. :smile: You already added it in the `package.json` file.

File: README.md (hunk 1 above)
Comment (silviubogan) [inline README.md]: @IonicaBizau, I corrected this in this commit: d103bc5. :smiley:

**Comment by IonicaBizau** [comment]
LGTM! :+1: Thanks.

**Pull request merged**